src/DependencyInjection/MongoDBExtension.php

```diff
@@ -70,7 +72,7 @@ private function createClients(string $defaultClient, array $clients, ContainerB
             $clientDefinition->setArgument('$uri', $configuration['uri']);
             $clientDefinition->setArgument('$uriOptions', $configuration['uriOptions'] ?? []);
             $clientDefinition->setArgument('$driverOptions', $configuration['driverOptions'] ?? []);
-
+            $clientDefinition->addTag('mongodb.client', ['name' => $client]);
```

Comment:
This name is used by `DataCollectorPass` and displayed in the profiler panel.
The tag didn't need to be in the service prototype.